care/emr/api/viewsets/report/template.py

```
@@ -0,0 +1,195 @@
+            )
+        else:
+            facility = instance.facility
+
+        if facility:
+            slug = Template.calculate_slug_from_facility(facility, instance.slug_value)
+        else:
+            slug = Template.calculate_slug_from_instance(instance.slug_value)
+
+        queryset = queryset.filter(slug__iexact=slug)
+        if queryset.exists():
+            raise ValidationError("Slug already exists.")
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)